docs/packages/xstate-test/index.md

```diff
@@ -173,6 +173,29 @@ const plans = todosModel.getShortestPathPlans({
 });
 ```
 
+### `testModel.getShortestPathPlansTo(options?)`
+
+Returns an array of testing plans based on the shortest paths from the test model's initial state to a given state specified by the `stateValue`.
+
+**Options**
+
+| Argument | Type     | Description                                                                                                    |
+| -------- | -------- | -------------------------------------------------------------------------------------------------------------- |
+| `stateValue` | string or function | Specifies a target state or a filter function which takes in the `state` and returns `true` if the state should be targeted |
```

Comment:
Thoughts on...

```
| `targetState` | string or function | Specifies a target state or a filter function which iterates over each path and returns `true` if the final state of the path matches the targetState string or filter"
```

My suggestion is by no means flawless.

Whatever change we make here should be reflected below.

Reply:
The current text is fine 👍